### ISSUE TITLE
spore server flaky

### ISSUE THREAD

**Issue by marcinwyszynski** [spore-sh/spore-cli-node#1]
<img width="260" alt="screen shot 2015-09-11 at 10 58 00" src="https://cloud.githubusercontent.com/assets/571786/9810910/228d6974-5874-11e5-9390-4ca397f8f4a5.png">

**Comment by treygriffith** [comment]
Has it ever worked for you? And is `spored` running? You can check the `spored` logs at `~/.spore/spored/log.log` and `~/.spore/spored/error.log`.

**Comment by marcinwyszynski** [comment]
Yes, it just took a few retries. I did indeed make sure the daemon was running. [Here](https://gist.github.com/marcinwyszynski/a89e17876a0b99a00a6e) is the error log - it doesn't say much more beyond what the screenshot says, though.

I mostly wanted to make sure that there is commitment to support the whole thing as my company has reached a point where we need to sort out configuration management and we're weighing our options.

**Comment by treygriffith** [comment]
Yeah that error log isn't too helpful. I took a look at the pod logs and didn't see anything unusual - my suspicion is that it was an issue with the hosting provider. I've opened a support ticket to see if I can track down what on their end might have been causing the timeouts. Can you provide me with some approximate times you were seeing timeouts? For now, I'm using `2015-09-11T08:59:17Z`, which is when you opened this issue.

There is definitely commitment to support this thing. I'm happy to help in any way that I can to help you get up and running. I'd also be willing to work out a specific arrangement with your company if that would make folks feel more comfortable.

I'll update this ticket when I get more info from the hosting provider. In the meantime, you can also reach me over email: hello@spore.sh
